public/src/js/profile.js

```diff
@@ -6,3 +6,11 @@ function populateProfile() {
         }
     })
 }
+
+function getFirestoreData(){
+    var output = document.getElementById('forData');
+    output.innerText = "test";
+    console.log("test")
```

Comment:
Is this testing code or is it intentional?

Reply:
@ANDREYDEN its testing code- please se commit message